src/path_info.cpp

```diff
@@ -158,6 +158,12 @@ void PATH_INFO::set_standard_filenames(void)
     update_pathname("obsolete-mods", FILENAMES["moddir"] + "obsolete-mods.json");
     update_pathname("defaultsounddir", FILENAMES["datadir"] + "sound");
 
+    // User's data directories
+    update_pathname("user_datadir", FILENAMES["user_dir"] + "data/");
+    update_pathname("user_gfxdir", FILENAMES["user_datadir"] + "gfx/");
```

Comment:
The default gfx dir lives under <user_dir>/gfx, not <user_datadir>/gfx, see line 122 in this method.  This will break your check in build_tilesets_list() and also cause the creation of a phantom directory in the default install.